content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/1. If-Else.md

```diff
@@ -0,0 +1,98 @@
+
+In het onderstaande voorbeeld is een variabel `age` aanwezig. Dit is de leeftijd van de student.
+Als de waarde van het variabel `age` groter of gelijk is aan `18` moet de student toegang krijgen.
+Als de waarde van het variabel `age` niet voldoet aan de conditie, dan gaat het programma naar de `else` statement.
+```C#
+int age = 16;
+if (age >= 18){
+    Console.WriteLine("Toegang verleend tot de content.");
+}
+else {
+    Console.WriteLine("Toegang geweigerd. Leeftijdsrestrictie: 18+");
+}
+```
+
+## Wat is een If-Else-If-structuur?
+Een *If-Else-If* structuur biedt een uitbreiding op de eenvoudige *If-Else*-structuur. Het maakt het mogelijk om meerdere condities achter elkaar te controleren. Wanneer een conditie waar (true) is, voert het programma de corresponderende code uit en negeert het de overige condities. Dit stelt de programmeur in staat om een serie aan beslissingen in een logische volgorde te plaatsen zonder herhaald gebruik van geneste *If-Else*-structuren, waardoor de code overzichtelijker en beter leesbaar blijft.
+
+## Hoe zit een If-Else-If-structuur in elkaar?
+- Het programma controleert de eerste conditie in de `if`-clausule.
+- Als de conditie niet waar is, gaat het programma verder naar de volgende conditie in de `else if`-clausule.
+- Als de `else if`-clausule ook niet waar is, dan wordt de `else`-clausule bereikt indien deze aanwezig is. Deze fungeert hier als een vangnet voor de overige gevallen.
+
+Bijvoorbeeld:
+```C#
+if (conditie_1){
+	// code om uit te voeren als de eerste conditie waar is.
+}
+else if(conditie_2){
+	// code om uit te voeren als de tweede conditie waar is.
+}
+else{
+    // alternatieve code als de conditie niet waar is.
+}
+```
+  Dit model toont de basisopbouw van de *If-Else*-logica.
+
+## Hoe gebruik je een If-Else-If-structuur?
+Hieronder zie je een voorbeeld van een *If-Else-If* structuur die een verschillende boodschap weergeeft op basis van de leeftijd van een bezoeker:
+
+```C#
+int age = 16;
+
+if (age >= 65) {
+    Console.WriteLine("Senioren korting toegepast.");
+}
+else if (age > 18) {
+    Console.WriteLine("Toegang verleend tot de content.");
+}
+else if (age >= 13) {
+    Console.WriteLine("Toegang verleend tot content voor jongeren.");
+}
+else {
+    Console.WriteLine("Toegang geweigerd. Leeftijdsrestrictie: 13+");
+}
+```
+
+**In dit voorbeeld:**
+- Als de gebruiker 65 jaar of ouder is, wordt de boodschap "Senioren korting toegepast." weergegeven.
+- Als de gebruiker ouder is dan 18 maar jonger dan 65, wordt "Toegang verleend tot de content." weergegeven.
+- Als de gebruiker tussen de 13 en 18 jaar oud is, krijgt hij of zij een aangepaste toegang.
+- Voor gebruikers jonger dan 13 wordt de toegang geweigerd.
+
+Deze aanpak minimaliseert overbodige code en laat de structuur duidelijk meerdere beslissingen afhandelen.
```

Comment:
De gebruikte gedachtelijn die hierin schuilt wordt niet benoemd. 
Namelijk (kortgezegd), door eerst met het meest restrictieve geval te beginnen, en deze 'grens' uit te breiden, heb je mooi het verloop in jaren wat je weergeeft. 

... .... Wellicht dat dat de referentie/verwijzing naar aanpakkennis is? "Welke condities zet je nu onder welke andere condities, waarom wel/niet?"